### ISSUE TITLE
Incorrect MAC addressed sorting

### ISSUE THREAD

**Issue by revolter** [alexandru-dinu/obsidian-sortable#35]
For these entries:

```
02-E5-27-38-DB-CF
00-55-DA-52-FA-15
1C-69-7A-6B-15-9D
```

this extension sorts them as:

```
00-55-DA-52-FA-15
1C-69-7A-6B-15-9D
02-E5-27-38-DB-CF
```

instead of:

```
00-55-DA-52-FA-15
02-E5-27-38-DB-CF
1C-69-7A-6B-15-9D
```

**Comment by alexandru-dinu** [comment]
Hello,

Apologies for the inconvenience.
As mentioned in the README, Sortable currently supports numbers, strings, [ISO dates](https://regex101.com/r/RfMAcx/1). 

I am rethinking the support for multiple data types, but in the meantime, if you are willing to contribute a fix, feel free to do so!

**Comment by revolter** [comment]
> As mentioned in the README, Sortable currently supports numbers, strings, [ISO dates](https://regex101.com/r/RfMAcx/1).

Yes, I got that. But what I don't understand is that even sorting them as simple strings should work. You can verify it using a tool like https://texttools.org/sort-text.

---

Actually, isn't it because you're enabling `numeric` here:

https://github.com/alexandru-dinu/obsidian-sortable/blob/cbdfea4f71d9358c2dad52954d2904543434a263/src/sortable.ts#L112

?

**Comment by alexandru-dinu** [comment]
Yes, it's due to the collator used for the natural sorting which interprets `02` > `1C`.
